djangocms_snippet/templates/djangocms_snippet/admin/change_form.html

```diff
@@ -19,12 +19,18 @@
 
     // init editor with settings
     var editor = ace.edit(div[0]);
-    editor.setTheme('ace/theme/' + settings.theme);
+    if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
```

Comment:
@crydotsnake I've "played" with dark mode some more and now am proposing a global toggle in the cms toolbar (see https://github.com/django-cms/django-cms/pull/7329). Can I propose to update this media query as follows:


            var darkMode;
            if (window.parent.CMS.API.Toolbar.get_color_scheme) {
                darkMode = window.parent.CMS.API.Toolbar.get_color_scheme() === 'dark';
            } else {
                // django CMS pre-3.11
                darkMode =  window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches;
            }
            if (darkMode) {
                editor.setTheme('ace/theme/tomorrow_night');
            }  else {
                editor.setTheme('ace/theme/github');
            }